src/sentry/static/sentry/app/utils/discover/eventView.tsx

```diff
@@ -259,6 +260,7 @@ class EventView {
     yAxis: string | undefined;
     display: string | undefined;
     interval?: string;
+    expired: boolean | undefined;
```

Comment:
You can make this optional like this so the functions below don't have to explicitly pass `undefined` to this where it is not necessary like with the URL.

```suggestion
    expired?: boolean;
```